portfolio/src/main/webapp/index.html

```diff
@@ -49,23 +51,40 @@ <h1 class="center">Jamie Tucker's Portfolio</h1>
           I may be having, while gaining more muscle mass for myself.
         <p>Click <button onclick="addRandomFact()">here</button> to see random facts about me!
         <div id="fact-container"></div>
-        <form action="/data" method="POST">
-          <p><b>Want to Leave a Comment?</b>
-          <br/>
-          <label for="name">Name:</label>
-          <input type="text" name="name">
-          <br/>
-          <label for="email">Email:</label>
-          <input type="email" name="email">
-          <br/>
-          <label for="comment">Comment:</label>
-          <textarea name="comment"></textarea>
-          <br/><br/>
-          <input type="submit" />
+        <hr id="divider">
+        <form class="requiresauth" action="/sports" method="POST">
```

Comment:
For comments you didn't reload the page, but it looks like you are for voting. Want to take a swing at making this refresh without reloading?